### ISSUE TITLE
Alt text display on Android is missing overflow indicator 

### ISSUE THREAD

**Issue by scottdotweb** [bluesky-social/social-app#7458]
### Steps to Reproduce

On Android, when image alt text is longer than two lines, no "..." indicator is shown, making it look like there isn't any further text. [Example post.](https://bsky.app/profile/james.claims/post/3lfllhdvres2r)

### Attachments

Android

![Screenshot_20250115_131404_Bluesky](https://github.com/user-attachments/assets/93fa4ff5-e3f4-4fe6-a704-74c9c5029c5d)

Mobile web

![Screenshot_20250115_131534_Firefox](https://github.com/user-attachments/assets/b57e151e-cb0c-4348-9d6c-9c26bb983328)

### What platform(s) does this occur on?

Android

### What version of the app are you using?

1.96.2.542
